gateway/event_handler_log.go

```diff
@@ -0,0 +1,65 @@
+package gateway
+
+import (
+	"fmt"
+	"github.com/TykTechnologies/tyk/apidef"
+	"github.com/TykTechnologies/tyk/config"
+	"github.com/TykTechnologies/tyk/internal/event"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// EH_LogHandler is an alias maintained for backwards compatibility.
+	// It is used to register log handler on an event.
+	EH_LogHandler = event.LogHandler
+)
```

Comment:
EH_LogHandler isn't used in file scope; event_system_test.go uses it, which ideally leaves this in event_system.go or const.go